app/controllers/lead_providers/your_schools_controller.rb

```diff
@@ -6,11 +6,14 @@ class YourSchoolsController < ::LeadProviders::BaseController
 
     def index
       @cohorts ||= @lead_provider.cohorts
-      @selected_cohort = if params[:selected_cohort_id]
-                           @cohorts.find(params[:selected_cohort_id])
-                         else
-                           @cohorts.find_by(start_year: Time.zone.today.year)
-                         end
+
+      # Don't break old URLs
+      if params[:selected_cohort_id]
+        redirect_to cohort: params[:selected_cohort_id]
+        return
+      end
+
+      @selected_cohort = @cohorts.find_by(start_year: params[:cohort] || Time.zone.today.year)
```

Comment:
I know I know, it wasn't you... Still can we make use of `Cohort.current` instead of the current year? This will break in January